endpoints/info/bidders_test.go

```diff
@@ -19,88 +19,60 @@ func TestPrepareBiddersResponseAll(t *testing.T) {
+			expected:     `[]`,
 		},
 		{
-			name:                "core-one-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledCore, "a": enabledCore},
-			givenRequestAliases: nil,
-			expected:            `["a","z"]`,
+			name:         "core-one-enabled",
+			givenBidders: config.BidderInfos{"a": enabledCore},
+			expected:     `["a"]`,
 		},
 		{
-			name:                "alias-one",
-			givenBidders:        config.BidderInfos{"a": enabledAlias},
-			givenRequestAliases: nil,
-			expected:            `["a"]`,
+			name:         "core-one-disabled",
+			givenBidders: config.BidderInfos{"a": disabledCore},
+			expected:     `["a"]`,
 		},
 		{
-			name:                "alias-mixed",
-			givenBidders:        config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
-			givenRequestAliases: nil,
-			expected:            `["a","b","c","d"]`,
+			name:         "core-many-mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": enabledCore},
+			expected:     `["a","b"]`,
 		},
 		{
-			name:                "alias-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledAlias, "a": enabledCore},
-			givenRequestAliases: nil,
-			expected:            `["a","z"]`,
+			name:         "core-many-sorted",
+			givenBidders: config.BidderInfos{"z": enabledCore, "a": enabledCore},
+			expected:     `["a","z"]`,
 		},
 		{
-			name:                "defaultrequest-one",
-			givenBidders:        config.BidderInfos{"a": enabledCore},
-			givenRequestAliases: map[string]string{"b": "a"},
-			expected:            `["a","b"]`,
+			name:         "alias-one",
+			givenBidders: config.BidderInfos{"a": enabledAlias},
+			expected:     `["a"]`,
 		},
 		{
-			name:                "defaultrequest-mixed",
-			givenBidders:        config.BidderInfos{"a": enabledCore, "b": disabledCore},
-			givenRequestAliases: map[string]string{"x": "a", "y": "b"},
-			expected:            `["a","b","x","y"]`,
+			name:         "alias-many-mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["a","b","c","d"]`,
 		},
 		{
-			name:                "defaultrequest-mixed-sorted",
-			givenBidders:        config.BidderInfos{"z": enabledCore},
-			givenRequestAliases: map[string]string{"a": "z"},
-			expected:            `["a","z"]`,
+			name:         "alias-many-sorted",
+			givenBidders: config.BidderInfos{"z": enabledAlias, "a": enabledCore},
+			expected:     `["a","z"]`,
 		},
 		{
-			name:                "mixed",
-			givenBidders:        config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
-			givenRequestAliases: map[string]string{"z": "a"},
-			expected:            `["a","b","c","d","z"]`,
+			name:         "mixed",
+			givenBidders: config.BidderInfos{"a": disabledCore, "b": disabledAlias, "c": enabledCore, "d": enabledAlias},
+			expected:     `["a","b","c","d"]`,
```

Comment:
I think you can delete this test case. It is the same as the `alias-many-mixed` case on lines 57-59.

Reply:
Differentiated.